_Group Projects/Group_5/demonstration.do

```diff
@@ -0,0 +1,14 @@
+clear
+
+global wd "/Users/miglepetrauskaite/Desktop/GITHUB/Repo/econ490-fall22/_Group Projects/Group_5"
+
+do "$wd/program.do"
+
+// demonstrating the use: regressing wage and age
+group5 ln_wage age
+
+// demonstrating  the use: regressing wage and belonging to a union
+group5 ln_wage union
+
+// another use:
```

Comment:
Could you clarify what this use does?

Reply:
It's a regression to see the effect of our chosen independent variable (belonging to a union) on the dependent variable (wages).